www/pages/docs/markdown.md

```diff
@@ -9,29 +9,29 @@ linkheadings: 3
 ## Markdown
 In this section we'll cover some of the Markdown related feature of Greenwood, which by default supports the [CommonMark](https://commonmark.org/help/) specification.
 
-
 ### Syntax Highlighting
-When rendering code fencing, if you add the word `render` before the language, the included [prismjs](https://prismjs.com/) library will add syntax highlighting.
+When rendering code fencing, if you add the language after the fencing, the included [prismjs](https://prismjs.com/) library will add syntax highlighting.
 
 e.g. use:
 
-```
-\`\`\`render js
+````js
+\`\`\`js
 const hello = "world";
 
 <p>\${hello}</p>
 \`\`\`
-```
+````
 
 To get the result:
 
-
-```render js
+```js
 const hello = "world";
 
-<p>${hello}</p>
+<p>\${hello}</p>
 ```
 
+> **Note:** As demonstrated in the above example, backticks and `\$` characters require `\\` to escape correctly.
```

Comment:
As pointed out, we've added this caveat to the documentation, to clarify the only exception to regular markdown that we're aware of.